### ISSUE TITLE
have a way to configure slack updates with this automator

### ISSUE THREAD

**Issue by joeycozza** [fs-webdev/github-automator#36]

**Comment by skye2k2** [comment]
Yeah. Fun problem. For manual releases, we could just have an optional parameter for slack alerts, along with the webhook url to post through. 

Tree currently utilizes this: https://github.com/fs-webdev/fs-common-build-scripts/blob/master/bin/report_failure_details.sh

Search currently utilizes this: https://github.com/fs-webdev/search-artifact-components/blob/master/bin/tag-release.js#L239
